### ISSUE TITLE
Authentication failed for recordings URI.

### ISSUE THREAD

**Issue by TiMeJuMp** [Azure-Samples/cognitive-services-speech-sdk#1828]
**Describe the bug**
Since yesterday (07.02.2023 ~1p.m CET ) we have trouble with the Speech SDK. We get the error message, that the presigned URL which we generate via S3 buckets is not valid, and the azure trancscripe-service could not access the file.

 "error": {
            "code": "InvalidUri",
            "message": "Authentication failed for recordings URI."
}

1. Create S3 bucket
2. Put in wav-file
3. Create presignedUrl
4. via Postman: send file via postman to azure https://westeurope.api.cognitive.microsoft.com/speechtotext/v3.1/transcriptions
``` 
{  
  "contentUrls": [
    "{{YOUR_FILE_LINK_HERE}}"
  ],
  "locale": "de-DE",
  "displayName": "Test1234",
  "description": "Test",
  "customProperties": {},
  "lastActionDateTime": "2023-02-08 15:35",
  "status": "NotStarted",
  "createdDateTime": "2023-02-08 15:35"
} 
``` 
5. Ask for transscription via provided URL in Response-body
6. Get error Message 
``` 
"error": {
            "code": "InvalidUri",
            "message": "Authentication failed for recordings URI."
        }
```
**Version of the Cognitive Services Speech SDK**
Which version of the SDK are you using. V3.1, but also 3.0 throws same error

**Comment by pankopon** [comment]
@zhouwangzw Please check.

**Comment by TiMeJuMp** [comment]
Hi @pankopon & @zhouwangzw,

we have also opened a ticket on azure for that issue and got already the answer. See below.

"Hello @Becker, Christian

We had issues with HEAD request in REST API. It was especially impacting non-Azure storage. When investigating we found out that the issues started immediately after our original code update was deployed. The issue results in an authentication error which is a very common client error. We fixed the issue by deploying a hot fix.

Looking forward to your feedback!
"

The error was faced between 08. Feb 1p.m. – 09. Feb 4 p.m CET. 

Think that issue can now be closed. 

Thanks Christian
